bluepyopt/ephys/locations.py

```diff
@@ -323,6 +323,99 @@ def __str__(self):
             self.soma_distance, self.seclist_name)
 
 
+class NrnSecSomaDistanceCompLocation(Location, DictMixin):
```

Comment:
Would there be a possibility to reuse code of the class above?

Reply:
I split the function instantiate from the class NrnSecSomaDistanceCompLocation to be able to reuse the second half of it in the new class.